### PR TITLE
Tap to focus issue.

### PR DESCRIPTION
**Pull request by saugat135**
Fix the issue where app crashed while tapping the camera preview for focus. #25

### CODE REVIEW

File: Example/FusumaExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -305,6 +305,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				INFOPLIST_FILE = FusumaExample/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				ONLY_ACTIVE_ARCH = NO;
 				PRODUCT_BUNDLE_IDENTIFIER = me.ytakzk.FusumaExample;
```
Comment (ytakzk) [inline Example/FusumaExample.xcodeproj/project.pbxproj:309]: @saugat135 Do we need this?

File: Fusuma.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -228,7 +228,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				MTL_ENABLE_DEBUG_INFO = YES;
-				ONLY_ACTIVE_ARCH = YES;
+				ONLY_ACTIVE_ARCH = NO;
 				SDKROOT = iphoneos;
```
Comment (ytakzk) [inline Fusuma.xcodeproj/project.pbxproj:232]: @saugat135 Do we need this?

File: Example/FusumaExample.xcodeproj/project.pbxproj (hunk 1 above)
Comment (leapfrog-saugat) [inline Example/FusumaExample.xcodeproj/project.pbxproj:309]: No we don't. I have the habit of changing this flag to No while compiling. Sorry for that.

**Comment by ytakzk** [comment]
@saugat135 Thank you!

**Pull request merged**